mmv1/third_party/terraform/resources/resource_dns_record_set.go

```diff
@@ -3,18 +3,16 @@ package google
 import (
 	"fmt"
 	"log"
-
-	"strings"
-
 	"net"
+	"strings"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"google.golang.org/api/dns/v1"
 )
 
 func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	o, n := d.GetChange("rrdatas")
-	if o == nil || n == nil {
+	if o == nil || n == nil || d.HasChange("routing_policy") {
```

Comment:
[note]
When I remove rrdatas field in tf files, `d.GetChange("rrdatas")` does not reflect its change...why?
So I added to check `d.HasChange("routing_policy")` as a workaround. (Because one of these must be set)
If you have a better way, please let me know.